api/envoy/extensions/filters/http/bandwidth_limit/v3alpha/bandwidth_limit.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.http.bandwidth_limit.v3alpha;
+
+import "google/protobuf/wrappers.proto";
+
+import "udpa/annotations/migrate.proto";
+import "udpa/annotations/status.proto";
+import "udpa/annotations/versioning.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.http.bandwidth_limit.v3alpha";
+option java_outer_classname = "BandwidthLimitProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).work_in_progress = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Bandwidth limit]
+// Bandwidth limit :ref:`configuration overview <config_http_filters_bandwidth_limit>`.
+// [#extension: envoy.filters.http.bandwidth_limit]
+
+message BandwidthLimit {
+  // Defines the mode for the bandwidth limit filter.
+  // Values represent bitmask.
+  enum EnableMode {
+    // Filter is disabled.
+    Disabled = 0;
```

Comment:
This should be pulled out into its own field using a RuntimeFeatureFlag. Here's an example of one:
https://github.com/envoyproxy/envoy/blob/853ecaa2f1e7364c1db49dad6a91bfcb3bcc4d61/api/envoy/config/listener/v3/quic_config.proto#L40

You can poke around and see how it would be used in the actual filter. This way folks can toggle it on and off for testing/rollouts/killswitching.

Reply:
sure, let me look into it.